Yashodha's org/Customer details API/service.bal

```diff
@@ -0,0 +1,38 @@
+import ballerina/http;
+import ballerinax/java.jdbc;
+import ballerina/io;
+
+# A service representing a network-accessible API
+# bound to port `9090`.
+
+type CustomerDetails record {
```

Comment:
Is it possible to make the record closed if the fields are limited to the mentioned ones?